packages/image_picker/ios/Classes/FLTImagePickerPhotoAssetUtil.m

```diff
@@ -57,11 +76,48 @@ + (NSString *)saveImageWithMetaData:(NSDictionary *)metaData
     data = [FLTImagePickerMetaDataUtil updateMetaData:metaData toImage:data];
   }
 
+  return [self createFile:data suffix:suffix];
+}
+
++ (NSString *)saveImageWithGIFInfo:(GIFInfo)info path:(NSString *)path {
```

Comment:
The meta data of the gif seems not copied?

Reply:
Certainly that was not done, so I fixed it.